test/index.js

```diff
@@ -231,7 +231,7 @@ describe('node-precinct', () => {
         amd: config.amd
       });
 
-      assert.deepEqual(stub.args[0][1], config.amd);
+      assert.deepEqual(stub.args[0][1].skipLazyLoaded, config.amd.skipLazyLoaded);
```

Comment:
Modifying these two tests to assert only on the specific property they're setting up, rather than the whole arguments objects, which now include more data irrelevant to this test, including `filename`.